andela/travel_tool_front
# #161379518 Remove auto-suggestion on date fields

#### What does this PR do?
Removes auto-complete from all input fields made by the FormsAPI.

#### Description of Task to be completed?
 - add `autoComplete: "off"` prop on the `renderInput()` method.

#### How should this be manually tested?
 - Clone the repo
    `git clone https://github.com/andela/travel_tool_front.git`
 - Switch to the project directory
   `cd travel_tool_front`
 - Checkout the **ch-remove-auto-suggest-on-date-161379518** branch
   `git checkout ch-remove-auto-suggest-on-date-161379518`
 - Install dependencies and start the server
   `make start`
 - Start the [backend](https://github.com/travel_tool_back) of this project.
 - Navigate to [travela](http://travela-local.andela.com:3000/) in your browser and log in with your Andela account. 
- Click on New Request to create a new travel request.
- You should notice that when you click on the date fields, there is no pop-up that shows you suggestions, only the date picker. (see the screenshots below)

#### Any background context you want to provide?
Ignore the .snap files in the files changed section.

#### What are the relevant pivotal tracker stories?
[#161379518](https://www.pivotaltracker.com/story/show/161379518)

#### Screenshots
![image](https://user-images.githubusercontent.com/41262970/47437318-02259200-d7b1-11e8-9387-247c5285761e.png)
![image](https://user-images.githubusercontent.com/41262970/47437328-05208280-d7b1-11e8-8451-3be7c79a217b.png)

#### Questions:
N/A
